awx/api/pagination.py

```diff
@@ -39,3 +56,17 @@ def get_html_context(self):
                                  for pl in context['page_links']]
 
         return context
+
+    def paginate_queryset(self, queryset, request, **kwargs):
+        self.count_disabled = 'count_disabled' in request.query_params
```

Comment:
The dashboard requests don't actually _need_ the count because they're not paging.  This cuts the time for the dashboard UJ and UJT requests **in half**.

Reply:
![image](https://user-images.githubusercontent.com/214912/64652673-6ca7e680-d3f2-11e9-8eef-e1192dbe4d79.png)

Reply:
I'm curious if there are any technical problems with linking the next / previous page or performing the slice for a given page number. I wouldn't think so, but DRF might be doing something unexpected. If a page doesn't exist, obviously that's a 404.

Reply:
Yea, if you disable the count and give a page number, you get a reasonable error:

<img width="436" alt="image" src="https://user-images.githubusercontent.com/214912/64665433-8d873080-d420-11e9-92e4-458adc40497a.png">